<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormINFO</class>
 <widget class="QWidget" name="FormINFO">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>260</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>260</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CDAudio</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cdtitleE">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="cdartistE">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>CD title:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>CD artist:</string>
   </property>
  </widget>
  <widget class="QLabel" name="chnL">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>141</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>mono/stereo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="artistE">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="titleE">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>231</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>51</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Artist:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>51</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Title:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cdgenereE">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>CD genere:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>CD year:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cdyearE">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Track:</string>
   </property>
  </widget>
  <widget class="QLabel" name="tL">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>31</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CD:</string>
   </property>
  </widget>
  <widget class="QLabel" name="cdL">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>/dev/null</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>closeButton</tabstop>
  <tabstop>cdtitleE</tabstop>
  <tabstop>cdartistE</tabstop>
  <tabstop>cdgenereE</tabstop>
  <tabstop>cdyearE</tabstop>
  <tabstop>artistE</tabstop>
  <tabstop>titleE</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
